test/images/scripts/lib/debian.install

```diff
@@ -46,6 +46,9 @@ if [ -n "$do_build" ]; then
     tar -xzf cockpit-*.tar.gz
     ( cd cockpit-*/
       cp -rp tools/debian debian
+      # Hack: Remove PCP build dependencies while pcp is not in testing
+      # (https://tracker.debian.org/pcp)
+      sed -i '/libpcp.*-dev/d' debian/control
```

Comment:
It seems this situation would prevent us from doing our weekly releases against Debian Testing? Broader question: Should we only be doing weekly releases against Jessie, and drop everything else?